library/ssl_cli.c

```diff
@@ -2188,6 +2188,7 @@ static int ssl_write_encrypted_pms( mbedtls_ssl_context *ssl,
     if( ssl->session_negotiate->peer_cert == NULL )
     {
         /* Should never happen */
+        MBEDTLS_SSL_DEBUG_MSG( 1, ( "should never happen" ) );
```

Comment:
Sorry, should have mentioned in my first comment: there are other occurrences of the same just below.

Reply:
Fixed.